src/test/java/seedu/address/model/person/TagsContainTagPredicateTest.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_PM;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_TP;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.tag.Tag;
+import seedu.address.testutil.TutorBuilder;
+
+public class TagsContainTagPredicateTest {
+    @Test
+    public void equals() {
+        List<Tag> firstPredicateKeywordList = Collections.singletonList(new Tag(VALID_TAG_PM));
+        List<Tag> secondPredicateKeywordList = Arrays.asList(new Tag(VALID_TAG_PM), new Tag(VALID_TAG_TP));
+
+        TagsContainTagPredicate firstPredicate = new TagsContainTagPredicate(firstPredicateKeywordList);
+        TagsContainTagPredicate secondPredicate = new TagsContainTagPredicate(secondPredicateKeywordList);
```

Comment:
Same as above, a suggestion for names is something like `oneKeywordPredicate` and `twoKeywordsPredicate`